minor cleanup updates to apm docs (#10050)
DataDog/documentation: content/en/tracing/setup_overview/setup/python.md

pip install ddtrace
```

> **Note:** This command requires pip version`18.0.0` or greater.  For Ubuntu, Debian, or another package manager, update your pip version with the following command:

Reviewer: ```suggestion
**Note:** This command requires pip version `18.0.0` or greater.  For Ubuntu, Debian, or another package manager, update your pip version with the following command:
```

> **Note:** This command requires at pip version `>=18.0.0`.  If you are using Ubuntu, Debian, or another package manager you may need to update your pip version using the below command.

Reviewer: ```suggestion
**Note:** This command requires pip version`18.0.0` or greater.  For Ubuntu, Debian, or another package manager, update your pip version with the following command:
```
